config/development.dist.yml

```diff
@@ -16,6 +16,7 @@ application:
   coc_link: http://confcodeofconduct.com/
   show_contrib_banner: true
   venue_image_path: /assets/img/venue.jpg
+  rating_system: YesNo
```

Comment:
If we use `PascalCase` for the value in configuration, maybe we can use this as well elsewhere?